src/test/java/no/nav/tag/finnkandidatapi/tilgangskontroll/TilgangskontrollServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package no.nav.tag.finnkandidatapi.tilgangskontroll;
+
+import no.nav.tag.finnkandidatapi.kandidat.Veileder;
+import no.nav.tag.finnkandidatapi.tilgangskontroll.veilarbabac.VeilarbabacClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TilgangskontrollServiceTest {
+
+    private TilgangskontrollService tilgangskontroll;
```

Comment:
Denne kan også annoteres med `@InjectMocks`

Reply:
https://tedvinke.wordpress.com/2014/02/13/mockito-why-you-should-not-use-injectmocks-annotation-to-autowire-fields/